test/SqlExtension.Tests/SqlOutputBindingTests.cs

```diff
@@ -40,5 +40,19 @@ public async Task TestAddAsync()
             };
             await collector.AddAsync(data);
         }
+
+        [Theory]
+        [InlineData("dbo.Products", "'dbo'", "'Products'")]
+        [InlineData("Products", "SCHEMA_NAME()", "'Products'")]
+        [InlineData("[dbo].[Products]", "'dbo'", "'Products'")]
+        [InlineData("[dbo].Products", "'dbo'", "'Products'")]
+        [InlineData("dbo.[Products]", "'dbo'", "'Products'")]
+        [InlineData("[Prod'ucts]", "SCHEMA_NAME()", "'Prod''ucts'")]
```

Comment:
Would suggest adding error cases (for example things that would make the parser fail). I assume having the fullname be something like `dbo.my'table` would be one of those (since to work the table name should be quoted)

Reply:
These can be a separate test if that's easier since it's slightly different logic to test that scenario